Source/RageCommunity.Library/Extensions/VectorExtensions.cs

```diff
@@ -116,5 +116,26 @@ public static bool IsOnScreen(this Vector3 position)
         {
             return NativeWrappers.GetHudScreenPositionFromWorldPosition(position, out float _, out float _);
         }
+        /// <summary>
+        /// Gets the nearest safe position to place a <see cref="Ped"/>
+        /// </summary>
+        /// <param name="positon">The position to check around</param>
+        /// <returns>If successfull, returns the <see cref="Vector3"/> result, otherwise returns <see cref="Vector3.Zero"/></returns>
+        public static Vector3 GetSafeCoordinateForPed(this Vector3 positon)
+        {
+            int[] flags = { 17, 1, 16 };
```

Comment:
Are you able to document what these flags mean?

Reply:
From: [https://docs.fivem.net/natives/?_0xB61C8E878A4199CA](https://docs.fivem.net/natives/?_0xB61C8E878A4199CA)
Flags are:
1 = 1 = B02_IsFootpath
2 = 4 = !B15_InteractionUnk
4 = 0x20 = !B14_IsInterior
8 = 0x40 = !B07_IsWater
16 = 0x200 = B17_IsFlatGround